marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/flowRunner.sjs

```diff
@@ -176,6 +181,20 @@ function invokeBeforeMain(stepExecutionContext, contentArray) {
   }
 }
 
+/**
+ * Applies source collections to written documents
```

Comment:
Because the code uses the term "originalCollections", we don't want to introduce "source collections" as a different term, as that raises the question of - What's the difference between source collections and original collections? 

The comment isn't quite accurate either - these aren't written documents yet, it's just an array of content objects. 

What the function is doing is - for each content object, it's concatenating original collections to the content object's collections if original collections exists. But if you look at the code, the code is basically saying that. So I don't think any comment is needed, the code reads well enough.